.github/workflows/pr_verification.yml

```diff
@@ -26,14 +27,26 @@ jobs:
         shell: bash
         run: |
           echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+          
+      - name: Install git
+        shell: bash
+        run: dnf --setopt install_weak_deps=False install -y git
```

Comment:
there is already a 'dnf install' step later on. can't we re-use that one? or if it needs to be before the `checkout@v3` step, then move it before it, and add git to the list of packages to install?

Reply:
yes, it just need to be before checkout, because if git is not installed it will downoald just files and it is not repository